pymess/models/emails.py

```diff
@@ -24,6 +26,18 @@
 )
 
 
+def generate_filename(instance, filename):
+    return 'pymess/e-mails/content_{}.txt'.format(uuid4())
+
+
+class EmailMessageQuerySet(MessageQueryset):
+
+    def create(self, content, **kwargs):
+        message = super().create(**kwargs)
+        message.content_file.save(None, ContentFile(content.encode()))
```

Comment:
New e-mails will have content saved in a file.

Reply:
You can set the file befor super().create. Only one db request will be required

`message = self.model(**kwargs)`
`message.content_file.save(None, ContentFile(content.encode()))`

Reply:
Implemented. This way, I am even able to make content file required later. Thanks.